pkgs/development/tools/godot/3/mono/make-deps.nix

```diff
@@ -0,0 +1,39 @@
+{ godot-mono, nuget-to-nix }:
+
+godot-mono.overrideAttrs (self: base: {
+  pname = "godot-mono-make-deps";
+
+  nativeBuildInputs = base.nativeBuildInputs ++ [ nuget-to-nix ];
+
+  shouldConfigureNuget = false;
+
+  outputs = [ "out" ];
+  buildPhase = " ";
+  installPhase = ''echo "No output intended. Run make-deps.sh instead." > $out'';
+
+  makeDeps = ''
+    set -e
+    outdir="$(pwd)"
+    wrkdir="$(mktemp -d)"
```

Comment:
Please add a trap to clean this up if it is run outside the sandbox

Reply:
I haven't really used traps before.  I'm uncertain what signals to handle in a situation like this.  Should I just handle 1, 2, 3, and 6?  Maybe 15?  What do you suggest?

Reply:
never mind, found a stack overflow suggestion: `trap 'rm -rf -- "$MYTMPDIR"' EXIT`